src/test/ERC1155.t.sol

```diff
@@ -442,7 +442,7 @@ contract ERC1155Test is DSTestPlus, ERC1155TokenReceiver {
 
         token.mint(from, 1337, 70, "");
 
-        hevm.prank(from);
+        hoax(from);
```

Comment:
why do we need hoax instead of vm.prank again?

Reply:
haha fair i think i'd prefer to prank unless we need hoax to avoid confusion

Reply:
yep will use prank :)